docs/notebooks/2_getting_started.ipynb

```diff
@@ -1072,11 +1085,14 @@
    "source": [
     "#### Horizon coordinate and time dimension\n",
     "\n",
-    "Even if no `interval` is called, `xs.climatological_mean` will substantially change the nature of the `time` dimension, because it now represents an aggregation over time. While no standards exist on how to reflect that in a dataset, the following was chosen for `xscen`:\n",
+    "Even if no `stride` is called, `xs.climatological_op` will substantially change the nature of the `time` dimension, because it now represents an aggregation over time. While no standards exist on how to reflect that in a dataset, the following was chosen for `xscen`:\n",
     "\n",
     "- `time` corresponds to the first timestep of each temporal average.\n",
     "- `horizon` is a new coordinate that either follows the format YYYY-YYYY or a warming-level specific nomenclature.\n",
-    "- The `cat:frequency` and `cat:xrfreq` attributes remain unchanged."
+    "- The `cat:frequency` and `cat:xrfreq` attributes remain unchanged.\n",
+    "\n",
+    "\n",
+    "Alternatively, setting the `horizons_as_dim` argument to *True* will rearrange the dataset with a new dimension `period` and a dimension named according to the temporal aggregation (`year`, `season` or `month`). The time stamps are conserved in the `time` coordinate as an array with those two new dimensions."
```

Comment:
```suggestion
    "Alternatively, setting the `horizons_as_dim` argument to *True* will rearrange the dataset with a new dimension `horizon` and a dimension named according to the temporal aggregation (`year`, `season` or `month`). The time stamps are conserved in the `time` coordinate as an array with those two new dimensions."
```

Reply:
`and a dimension named according to the temporal aggregation (`year`, `season` or `month`)`

If `year` has been removed, it would need to be removed here too.